src/components/ResourceCard/ResourceCard.js

```diff
@@ -75,33 +76,53 @@ const ResourceCard = ({
 }) => {
   const resourceDate = getResourceDate(resource);
   const firstCardStyle = index === 0 ? styles.firstCard : {};
+  const displayType = SINGULAR_RESOURCE_TYPES[resource.type];
 
-  return (
-    <View style={[styles.root, firstCardStyle]}>
-      <View style={styles.header}>
+  const cardHeader = () => {
+    if (fromDetailsPanel) {
+      return (
+        <>
+          <BaseText style={styles.typeLabel}>{displayType}</BaseText>
+          <View style={styles.rightIconsContainer}>
+            <BaseText style={styles.resourceDate}>{resourceDate}</BaseText>
+            <CollectionIcon
+              showCount={false}
+              collectionId={collectionId}
+              resourceIds={[resourceId]}
+            />
+          </View>
+        </>
+      );
+    }
```

Comment:
this should probably be a separate `CardHeader` component (could be in same file)

* function body is not re-evaluated on every render
* more explicit
* you do not need to reason about the `ResourceCard` variables that are closed-over